Sources/StreamChatUI/ChatMessageList/ChatMessage/GiftPacketSentBubble.swift

```diff
@@ -289,10 +289,10 @@ class GiftBubble: UITableViewCell {
         let bigInt = BigUInt(amount) {
             if extraData.tokenAddress != nil {
                 let double = amountFromWeiUnit(amount: bigInt, weiUnit: Constant.tokenWeiUnit)
-                amountLabel.text = "\(double) USDC".replacingOccurrences(of: ".0", with: "")
+                amountLabel.text = "\(NumberUtils.formatCurrency(double)) USDC"
```

Comment:
USDC is a token, not currency lika USD, so should use coin format (with 4 digits as default here)